new-pipeline/dist.html

```diff
@@ -24,12 +24,16 @@
             <form class="navbar-form inline pull-right">
                 <span class="permalink-control"></span>
                 <a href="https://github.com/mozilla/telemetry-dashboard/issues" class="btn btn-success" title="Report a bug in Telemetry dashboards" target="_blank"><i class="fa fa-bug"></i> Report bug</a>
-                <a href="./evo.html" class="btn btn-default" id="switch-views" title="Switch to the evolution view"><i class="fa fa-line-chart"></i> Evolution View</a>
+                <a href="/" class="btn btn-default btn-primary" title="Landing page for Telemetry dashboards"><i class="fa fa-home"></i> Dashboards Home</a>
```

Comment:
"Dashboards Home" was removed by another PR. This was likely readded as part of your merge.

Reply:
Oh sorry